src/Codeception/Subscriber/ErrorHandler.php

```diff
@@ -98,8 +98,10 @@ public function shutdownHandler()
         if (!$this->suiteFinished && (
             $error === null || !in_array($error['type'], [E_ERROR, E_COMPILE_ERROR, E_CORE_ERROR])
         )) {
-            throw new \RuntimeException('Command Did Not Finish Properly');
-        } elseif (!is_array($error)) {
+            echo "\n\n\nCOMMAND DID NOT FINISH PROPERLY.\n";
+            return;
```

Comment:
This would still look like successful run in CI system.
It would be better to use `die(1);` in order to set non-zero exist code.

Reply:
I was deceived by `executeFailCommand`: I thought it would check that the command fails, instead it only accept possible failing commands:

https://github.com/Codeception/Codeception/blob/feb566a9dc26993611602011ae3834d8e3c1dd7f/src/Codeception/Module/Cli.php#L52-L54

PR updated.

This should be valid also for the `FATAL ERROR. TESTS NOT FINISHED.`, shouldn't it?